geomesa-nifi-processors/geomesa-nifi-processors-accumulo/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm... how many log4j.xml files do we have:)?  That can be a later thing (obviously).

Reply:
;)